src/pages/registration/files_and_license_tab/FilesTableRow.tsx

```diff
@@ -275,18 +275,26 @@ export const FilesTableRow = ({
                           row
                           sx={{ flexWrap: 'nowrap' }}
                           onChange={(event) => {
-                            const fileVersion = event.target.value as FileVersion;
-                            setFieldValue(field.name, fileVersion);
+                            const newFileVersion = event.target.value as FileVersion;
+                            const previousFileVersion = field.value as FileVersion;
```

Comment:
Denne ser ut til å ha riktig type fra før, fra linje 268?
```suggestion
                            const previousFileVersion = field.value;
```